### PR TITLE
Article about JBoss Fuse Tooling new Global Configuration editor page

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+= JBoss Fuse Tooling - Support of Global configurations
```
Comment (xcoulon) [inline blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc]: FYI, you don't have to include the date in the filename anymore. Only the `page-date` metadata matters. This avoids renaming the file if it is published later than expected ;-)

File: blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+:page-date:2016-05-20
+:page-layout: blog
+:page-author: apupier
+:page-tags: [jbtis,fuse]
+
+Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/18/jboss-fuse-tooling-support-of-global-configurations/[Red Hat Developers Program blog]
+
+A new editor tab is now available to manipulate global configurations.
+
```
Comment (alexeykazakov) [inline blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc]: IMO some introduction part is missing.
Something like you have in the original article in RHD blog:

> Red Hat JBoss Fuse is an open source, lightweight and modular integration platform that allows you to connect services and systems across your entire application portfolio. And if you’re familiar with Fuse, you’re probably familiar with the Fuse Tooling that comes with Red Hat JBoss Developer Studio.
> 
> As I mentioned in an earlier post, the 8.0.0.Beta2 version of JBoss Fuse Tooling is now available. Apart from the diagram tooling rework, there is yet another new, awaited feature. You can find it in the new “Configurations” editor — designed to manipulate global configurations, i.e. elements defined at the Camel context scope. 

Something about where this new feature available and how you can get it to try.

Comment (apupier) [inline blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc]: I added the second part only.
From the link to "earlier post", it explains how to install it.

**Comment by alexeykazakov** [comment]
There is something wrong. I tried to build the site and run it locally. It fails.
See https://github.com/jbosstools/jbosstools-website for instructions how to run the site locally.

**Comment by xcoulon** [comment]
@alexeykazakov @apupier it works fine here, once I added the space to separate the date key and value in the metadate: `:page-date: 2016-05-20`  
The first line at the top of the `.adoc` file should not be blank, but that does not break the build.

**Comment by xcoulon** [comment]
@apupier also, you may want to add the `jbosscentral` tag to have your article featured on JBoss Central. 
@maxandersen which tag should we add to get the article included in JBoss.org's feed and planeteclipse.org's ?

**Comment by maxandersen** [comment]
planeteclipse should happen for any blog entry on tools.jboss.org/blog

**Comment by apupier** [comment]
@maxandersen this one http://tools.jboss.org/blog/2016-05-18-EclipseFuseTooling-DiagramReworked.html is not part of PalnetEclipse

@xcoulon I added the space and the tag jbosscentral

File: blog/EclipseFuseTooling-NewGlobalConfiguration.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
+
+= JBoss Fuse Tooling - Support of Global configurations
+
```
Comment (alexeykazakov) [inline blog/EclipseFuseTooling-NewGlobalConfiguration.adoc]: There should not be a new line.

File: blog/EclipseFuseTooling-NewGlobalConfiguration.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+= JBoss Fuse Tooling - Support of Global configurations
+
+:page-date: 2016-05-23
+:page-layout: blog
+:page-author: apupier
+:page-tags: [jbtis,fuse, jbosscentral]
+
+Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/18/jboss-fuse-tooling-support-of-global-configurations/[Red Hat Developers Program blog]
+
+As I mentioned in an earlier post[http://developers.redhat.com/blog/2016/05/17/jboss-fuse-tooling-diagram-reworked-new-shiny-colors-and-more/], the 8.0.0.Beta2 version of JBoss Fuse Tooling is now available. Apart from the diagram tooling rework, there is yet another new, awaited feature. You can find it in the new “Configurations” editor — designed to manipulate global configurations, i.e. elements defined at the Camel context scope.
```
Comment (alexeykazakov) [inline blog/EclipseFuseTooling-NewGlobalConfiguration.adoc]: Should be http://developers.redhat.com/blog/2016/05/17/jboss-fuse-tooling-diagram-reworked-new-shiny-colors-and-more/[post]

**Comment by alexeykazakov** [comment]
I fixed the problems mentioned above, rebased and pushed to master and production.

**Comment by apupier** [comment]
thank you @alexeykazakov

File: blog/EclipseFuseTooling-NewGlobalConfiguration.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (maxandersen) [inline blog/EclipseFuseTooling-NewGlobalConfiguration.adoc]: just noticed this - please try not to use mixed cases in docs used for urls. all lower case please ;)

**Comment by maxandersen** [comment]
about planet eclipse - the news feed registered at eclipse planet is http://tools.jboss.org/blog/news.atom so all our blog entries should eventually show up. If not, please open bug at eclipse bugzilla about their aggregator being broken somehow ;)

File: blog/EclipseFuseTooling-NewGlobalConfiguration.adoc (hunk 5 above)
Comment (maxandersen) [inline blog/EclipseFuseTooling-NewGlobalConfiguration.adoc]: another one - you no longer need to repeat the date in the file name - just have :page-date in the preamble.

**Pull request closed without merging**